src/Fabulous.MauiControls/Views/Cells/EntryCell.fs

```diff
@@ -70,7 +70,7 @@ module EntryCellBuilders =
             WidgetBuilder<'msg, IFabEntryCell>(
                 EntryCell.WidgetKey,
                 EntryCell.Label.WithValue(label),
-                EntryCell.TextWithEvent.WithValue(ValueEventData.create text (fun args -> onTextChanged args.NewTextValue |> box))
+                EntryCell.TextWithEvent.WithValue(ValueEventData.create text (fun _ -> onTextChanged))
```

Comment:
In Fabulous.Avalonia, I replaced `ValueEventData.create value (fun args -> fn args |> box)` with `ValueEventData.create value fn` because we were not accessing any property of the `args`.

But here, we are accessing the `NewTextValue` property, so we need to still write the lambda function and call it correctly.

Reply:
Should be fixed in the last commit :)